spec/services/auctions_closed_within_last_twenty_four_hours_finder_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe AuctionsClosedWithinLastTwentyFourHoursFinder do
+  describe '#perform' do
+    it 'returns auctions with an end datetime that occured today' do
+      Timecop.freeze(Time.parse("19:00:00 UTC")) do
+        _old_auction = create(:auction, ended_at: 2.days.ago)
+        newly_closed_auction = create(:auction, ended_at: 2.hours.ago)
+        _future_auction = create(:auction, ended_at: 2.days.from_now)
```

Comment:
Easier to make these hours arguments for a method? And the arguments we end passing in would be constants (as you suggested above)?